frontend/scss/components/organisms/component-visual.scss

```diff
@@ -25,7 +25,7 @@
   }
 
   &-link {
-    margin-top: 3em;
+    margin: 3em 7px 0;
```

Comment:
I know that's really picky, but can we use consistent units here? 😁

Reply:
fixed 😈